ml/src/main/scala/ai/h2o/sparkling/ml/params/HasPreTrained.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.ml.params
+
+import ai.h2o.sparkling.H2OFrame
+import org.apache.spark.sql.DataFrame
+
+trait HasPreTrained extends H2OAlgoParamsBase with HasDataFrameSerializer {
+  private val preTrained = new NullableDataFrameParam(
+    this,
+    "preTrained",
+    "Id of a data frame that contains a pre-trained (external) word2vec model.")
```

Comment:
Is the comment correct? :` Id of a data frame`

Reply:
that's copypasted from the [h2o-3 docs](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/word2vec.html?highlight=pre_trained):
`pre_trained: Specify the ID of a data frame that contains a pre-trained (external) Word2vec model.`
that's a bit confusing but I guess every h2o frame has an ID and that could be passed here, not sure what to do about it

Reply:
Can we remove at least: `Id of `? Each H2OFrame has its own key(id), which serves as a reference the H2O cluster DKV. SW API uses Spark dataframes instead of H2O frames. After SW transforms dataframe to h2o frame, it passes [its key to h2o backend](https://github.com/h2oai/h2o-3/blob/879e58d3e5f98064b738189febecd845bf4f87c4/h2o-algos/src/main/java/hex/schemas/Word2VecV3.java#L53), but this is hidden to users.

This description will be utilized as source for [generated documentation](https://docs.h2o.ai/sparkling-water/3.0/latest-stable/doc/parameters/parameters_H2OWord2Vec.html).